pykern/pkcli/test.py

```diff
@@ -18,8 +18,9 @@
 
 SUITE_D = "tests"
 _COROUTINE_NEVER_AWAITED = re.compile("RuntimeWarning: coroutine .+ was never awaited")
```

Comment:
This should capture the entire line. You'll want to use "re.MULTILINE" flag on the compile.